meshtastic/interfaces/ble/reconnection.py

```
@@ -94,6 +98,79 @@ def __init__(
+                ThreadLike,
+                valid_create_thread(
+                    target=target,
+                    args=args,
+                    kwargs=kwargs,
+                    name=name,
+                    daemon=daemon,
+                ),
+            )
+        raise AttributeError(
+            "Thread coordinator is missing create_thread/_create_thread"
+        )
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)